src/overlays/actors/ovl_En_Tanron3/z_en_tanron3.h

```diff
@@ -7,10 +7,53 @@ struct EnTanron3;
 
 typedef void (*EnTanron3ActionFunc)(struct EnTanron3*, GlobalContext*);
 
+typedef enum {
+    /* 0 */ TIMER_PICK_DIRECTION_OR_DIE,
+    /* 1 */ TIMER_OUT_OF_WATER,
+    /* 2 */ TIMER_ATTACK_OR_WAIT,
+    /* 3 */ TIMER_MAX
+} TimerWork;
+
+typedef struct {
+    /* 0x00 */ u8 unk_00;
+    /* 0x02 */ s16 unk_02;
+    /* 0x04 */ Vec3f unk_04;
+    /* 0x10 */ Vec3f unk_10;
+    /* 0x1C */ Vec3f unk_1C;
+    /* 0x28 */ char unk_28[0xC];
+    /* 0x34 */ Vec3f unk_34;
+    /* 0x40 */ char unk_40[0x4];
+} UnkTanron3Effect;
+
 typedef struct EnTanron3 {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x1B4];
-    /* 0x02F8 */ EnTanron3ActionFunc actionFunc;
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[10];
+    /* 0x1C4 */ Vec3s morphTable[10];
+    /* 0x200 */ s16 timer;
+    /* 0x202 */ u8 isPassive;
+    /* 0x203 */ u8 isBeached;
+    /* 0x204 */ s16 workTimer[3];
```

Comment:
```suggestion
    /* 0x204 */ s16 workTimer[TIMER_MAX];
```
Well, I guess that settles who wrote this...